tests/e2e/specs/in-game/one-offs/3_threes.spec.js

```diff
@@ -137,10 +137,14 @@ describe('Playing THREEs', () => {
     cy.resolveThreeOpponent(Card.ACE_OF_SPADES);
 
     cy.get('#waiting-for-opponent-resolve-three-scrim').should('not.exist');
-
+    cy.get('[data-cy="scrap-chosen-card"]').should('be.visible');
```

Comment:
Have you tried adding a not exists assertion after this? Without the 8 for points below
```suggestion
    cy.get('[data-cy="scrap-chosen-card"]').should('be.visible');
    cy.get('[data-cy="scrap-chosen-card"]').should('not.exist');
```